Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:20.04
-ARG VERSION=1.4.0
```

Comment:
Could you remove the line in `Cargo.toml` that updates this?

Reply:
updated